style.css

```diff
@@ -1,7 +1,399 @@
+}
+
+.tickets__paragraph {
+  color: #263b83;
+  font-weight: 900;
+  font-size: 30px;
+  margin: 0;
+  margin-bottom: 33px;
+}
+
+.tickets__button {
+  background-color: #ffda45;
+  border: 0;
+  box-shadow: 5px 5px 20px rgba(0, 0, 0, .25);
+  margin-bottom: 18px;
+}
+
+.tickets__reserve {
+  background-color: #e3eafe;
+  border-bottom: 3px solid #ffda45;
+  border-left: 0;
+  border-right: 0;
+  border-top: 0;
+  color: #585858;
+  font-size: 20px;
+  font-weight: 900;
+  margin-bottom: 55px;
+  padding-bottom: 13px;
+}
+
+@media screen and (max-width: 320px) {
+  .tickets {
+    align-items: center;
+    background-color: #e3eafe;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .title {
+    color: #585858;
+    font-weight: 500;
+    font-size: 20px;
+  }
+
+  .tickets__title {
+    align-self: center;
+    height: 70px;
+    margin: 0 35px 7px 35px;
+    text-align: center;
+    white-space: pre-wrap;
+  }
+
+  .tickets__paragraph {
+    align-self: center;
+    color: #263b83;
+    font-weight: 900;
+    font-size: 10px;
+    margin: 0;
+    margin-bottom: 19px;
+  }
+
+  .tickets__button {
+    background-color: #ffda45;
+    border: 0;
+    box-shadow: 5px 5px 20px rgba(0, 0, 0, .25);
+    margin-bottom: 39px;
+  }
+
+  .tickets__reserve {
+    background-color: #e3eafe;
+    border-bottom: 3px solid #ffda45;
+    border-left: 0;
+    border-right: 0;
+    border-top: 0;
+    color: #585858;
+    font-size: 20px;
+    font-weight: 900;
+    margin-bottom: 23px;
+    padding-bottom: 13px;
+  }
+}
+
+.participants {
+  align-items: center;
+  background-color: #405dbb;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 89px;
+}
+
+.wrap-info {
+  color: #fff;
+  /*flex-direction: column;*/
+  font-weight: 900;
+  margin-left: 130px;
+  margin-top: 118px;
+}
+
+.wrap-info__title {
+  margin-bottom: 20px;
+  margin-right: 246px;
+  margin-top: 0;
+}
+
+.wrap-info__paragraph {
+  margin-bottom: 108px;
+  margin-right: 43px;
+  margin-top: 0;
+}
+
+.wrap-info__pic {
+  height: 327px;
+  margin-right: 43px;
+  width: 464px;
+}
+
+.button__request {
+  margin-right: 131px;
+}
+
+@media screen and (max-width: 320px) {
+
+
+
+
+
+
+
+
+}
+.speakers {
+  display: flex;
+  flex-direction: column;
+}
+
+.speakers__title {
+  align-self: center;
+  margin-bottom: 78px;
+  margin-top: 0;
+}
+
+.photos {
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+
+  /*.basis {
+    background-color: #e3eafe;
+    font-family: "Roboto", serif;
+    font-weight: 900;
+    margin: 0;
+    max-height: 1635px;
+    max-width: 480px;
+    min-width: 300px;
+  }
+
+  .introduce {
+    color: #e1e6e9;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    font-weight: 900;
+    justify-content: center;
+  }
+
+  .introduce__title {
+    font-size: 30px;
+    align-self: center;
+  }
+}*/
+/*@media screen and (min-width: 320px) and (max-width: 480px) {
+  .wrap__title {
+    font-size: 20px;
+    background-color: blue;
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)